### ISSUE TITLE
E3002 - Template that is valid with 0.78.2 fails with E3002 in 0.79.3

### ISSUE THREAD

**Issue by wmartins** [aws-cloudformation/cfn-lint#2817]
### CloudFormation Lint Version

0.78.2 / 0.79.3

### What operating system are you using?

Mac

### Describe the bug

I have a template that was linted correctly using `cfn-lint@0.78.2`, but it started failing using `cfn-lint@0.79.3`.

It's failing with the following error:

```
E3002 Property is an object instead of String at Resources/RDSDBClusterParameterGroup/Properties/Family
/tmp/stack.yaml:29:7
```

### Expected behavior

Expected it to be a valid template, following the same behavior as the old version.

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Description: "CloudFormation Template to create Aurora Postgresql Cluster DB Instance"
Transform:
  - AWS::LanguageExtensions
  - AWS::Serverless-2016-10-31
Parameters:
  DBEngineVersion:
    Description: Select Database Engine Version
    Type: String
    Default: 12.8
    AllowedValues:
      - 12.8
      - 12.12
Mappings:
  DBFamilyMap:
    "11.13":
      "family": "aurora-postgresql11"
    "12.8":
      "family": "aurora-postgresql12"
    "12.12":
      "family": "aurora-postgresql12"
    "13.4":
      "family": "aurora-postgresql13"
Resources:
  RDSDBClusterParameterGroup:
    Type: AWS::RDS::DBClusterParameterGroup
    Properties:
      Description: !Join ["- ", ["Description ", !Ref 'AWS::StackName']]
      Family: !FindInMap [DBFamilyMap, !Ref DBEngineVersion, "family"]
      Parameters:
        rds.force_ssl: 1
        apg_ccm_enabled: 1
```
